packages/ember-toucan-form/src/components/toucan-form.ts

```diff
@@ -38,6 +38,18 @@ export interface ToucanFormComponentSignature<
           'form'
         >;
         Textarea: WithBoundArgs<typeof TextareaFieldComponent<DATA>, 'form'>;
+
+        /**
+         * Yielded action that will trigger form validation and submission, same as when triggering the native `submit` event on the form.
+         *
+         * Note that calling this directly is **not** required for most cases. The implementation only requires a button tag with the `type="submit"` attribute set; however, this is exposed for more complex cases.
+         */
+        submit: () => void;
+
+        /**
+         * Yielded action that will reset form state, same as when triggering the native `reset` event on the form.
```

Comment:
Should note that if a FormData is present, it will reset to that, not just blank form inputs.